samples/dlp_inspect_image_file.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright 2018, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+// [START dlp_inspect_image_file]
+// Imports the Google Cloud Data Loss Prevention library
+const DLP = require('@google-cloud/dlp');
+
+// Import other required libraries
+const fs = require('fs');
+
+/**
+ * Inspects the specified image file.
+ *
+ * @param projectId {string} Your Google Cloud Project ID.
+ * @param filepath {string} The path to the image file to inspect.
+ * @returns {object} The InspectContentResponse.
+ */
+export async function inspectImageFile(
+  projectId = 'YOUR_PROJECT_ID',
+  filepath = 'path/to/image.png'
+) {
+  // Instantiates a client
+  const dlp = new DLP.DlpServiceClient();
+
+  // Get the bytes of the file
+  const fileBytes = Buffer.from(fs.readFileSync(filepath)).toString('base64');
```

Comment:
This isn't how we would expect someone to load this file in production.  Can we instead do an async load here?

Reply:
Done